src/main/java/com/theelixrlabs/healthcare/service/DoctorPatientAssignmentService.java

```diff
@@ -0,0 +1,97 @@
+package com.theelixrlabs.healthcare.service;
+
+import com.theelixrlabs.healthcare.constants.DoctorConstants;
+import com.theelixrlabs.healthcare.constants.DoctorPatientAssignmentConstants;
+import com.theelixrlabs.healthcare.dao.DoctorPatientAssignmentRepository;
+import com.theelixrlabs.healthcare.dto.DoctorPatientAssignmentDTO;
+import com.theelixrlabs.healthcare.exceptionHandler.CustomException;
+import com.theelixrlabs.healthcare.model.DoctorPatientAssignmentModel;
+import com.theelixrlabs.healthcare.repository.DoctorRepository;
+import com.theelixrlabs.healthcare.repository.PatientRepository;
+import org.springframework.context.MessageSource;
+import org.springframework.stereotype.Service;
+
+import java.time.Instant;
+import java.util.Date;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+@Service
+public class DoctorPatientAssignmentService {
+
+    private final DoctorPatientAssignmentRepository doctorPatientAssignmentRepository;
+    private final PatientRepository patientRepository;
+    private final DoctorRepository doctorRepository;
+    private final MessageSource messageSource;
+
+    public DoctorPatientAssignmentService(DoctorPatientAssignmentRepository doctorPatientAssignmentRepository,
+                                          PatientRepository patientRepository, DoctorRepository doctorRepository,
+                                          MessageSource messageSource) {
+        this.doctorPatientAssignmentRepository = doctorPatientAssignmentRepository;
+        this.patientRepository = patientRepository;
+        this.doctorRepository = doctorRepository;
+        this.messageSource = messageSource;
+    }
+
+
+    private void validateDoctorPatientAssignmentDTO(DoctorPatientAssignmentDTO doctorPatientAssignmentDTO) throws CustomException {
```

Comment:
Rename validateDoctorPatientAssignmentDTO to validateDoctorPatientAssignmentDto

Reply:
Resolved

Reply:
@SambitKSahu99  If doing rename in one place follow the same in all similar occurrences. Is the DTO class name is ok DoctorPatientAssignmentDTO ? I suggest to rename as DoctorPatientAssignmentDto to keep consistency in all places.